src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/TokenCachePlugin.cs

```diff
@@ -57,14 +57,8 @@ public static void BeforeAccess(TokenCacheNotificationArgs args)
             }
             catch (Exception ex)
             {
-                if (LoggerCallbackHandler.PiiLoggingEnabled)
-                {
-                    CallState.Default.Logger.WarningPii(null, "Failed to load cache: " + ex.ToString());
-                }
-                else
-                {
-                    CallState.Default.Logger.Warning(null, "Failed to load cache: " + ex.PiiLessToString());
-                }
+                CallState.Default.Logger.Warning(null, "Failed to load cache: ");
+                CallState.Default.Logger.ErrorPii(null, ex);
```

Comment:
add "Failed to load cache" in the message like before

Reply:
that message is logged as Warning, add to what ?

Reply:
pii logs includes non pii logs , so what is the point of inclusion of "Failed to load cache" message into errorPii ? also ErrorPii accept only one parameter - exception

Reply:
this is needed because developer can choose to write PII logs in a separate location. I think MS office pushes PII logs to a separate and secure location with security parameters applied it/

Reply:
i see,  actually in that case they will have to duplicate all log messages - as pii and as no pii. I would say it might be better to handle this at call back level - so if app developer want to have separate pii and non pii logs, his log delegate implementation might do like: if message does contain pii write message to pii and non-pii logs, if message does contain pii - write it only to pii log